src/lang/en-US.ts

```diff
@@ -29,7 +29,7 @@ const CommonMessages: ResolvedIntlConfig['messages'] = {
     'common.loggedOut': `You have been logged out. Please log back in.`,
     'common.missing': `N/A`,
     'common.none': `none`,
-    'common.noUnDo': `This action cannot be undone.`,
+    'common.noUnDo': `All items will be permanently deleted and this action cannot be undone. Please review the list to continue.`,
```

Comment:
This piece of content is only passed to the `title` property of the `AlertBox` component (e.g., _src/components/tables/RowActions/Delete/Confirmation.tsx_). This _feels_ like a misuse of the property given it is more body copy than a title/header. Any opposition to displaying this content via a child of the `AlertBox` component?

Reply:
Why did we replace this title? As stated here - seems like this is much more of a body message and not a title. Seems we should keep the title simple and then use this as a child.

Reply:
Dave requested a change the content of this warning (see link to issue comment in PR description). Moving this to a child of the `AlertBox` component in the two instances where it is used.

Reply:
I feel like the requested change can be done while leaving a title. But whatever y'all decided is fine with me.

Reply:
If you are fine with unused content being left behind, then yes, this can be updated now. Will do.